src/legacy/core_plugins/kibana/public/discover/components/field_chooser/discover_field_search.tsx

```diff
@@ -18,7 +18,16 @@
  */
 import React from 'react';
 import { i18n } from '@kbn/i18n';
-import { EuiButtonIcon, EuiFieldSearch, EuiFlexGroup, EuiFlexItem, EuiToolTip } from '@elastic/eui';
+import {
+  EuiButtonIcon,
+  EuiFacetButton,
+  EuiFieldSearch,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiToolTip,
+} from '@elastic/eui';
```

Comment:
Please remove `EuiButtonIcon` and `EuiToolTip`, since they're not in use, that's causing the type check error.
You can run the type check locally by `node scripts/type_check`

Reply:
Sorry, I had this locally, just forgot to commit the file 🤦‍♀

Reply:
I can't count the times I've broken Jenkins heart by forgetting similar things. he was never mad at me :)